demos/django-todolist/lib/apiclient.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:convert';
```

Comment:
Can you use snake case for this file name i.e. api_client to conform to files names in the repo